docs/src/examples/optimization_with_complex_variables/power_flow_optimization.jl

```diff
@@ -18,7 +18,7 @@ W = ComplexVariable(n, n);
 objective = real(sum(diag(W)));
 c1 = Constraint[];
 for i in 2:n
-    push!(c1, sum(W[i, :] .* (Y[i, :]')) == inj[i])
+    push!(c1, sum(W[i, :] .* conj(Y[i, :])) == inj[i])
```

Comment:
Ooooofff. This is pretty subtle...

Reply:
Good catch! Btw maybe this could be `dot`? It used to be bugged in the complex case so maybe the original author didn’t use it

Reply:
Updated to use `dot`. Yeah, I assume the previous code was just trial and error to find what worked.